vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -264,12 +268,14 @@ function Install-ContainerD {
         Expand-Archive -path $containerdTmpDest -DestinationPath $installDir -Force
     } else {
         tar -xzf $containerdTmpDest --strip=1 -C $installDir
+        Get-ChildItem -Path $installDir\bin -Recurse -File | Move-Item -Destination $installDir
+        Remove-Item -Path $installDir\bin -Force
     }
     Remove-Item -Path $containerdTmpDest | Out-Null
 
-    $newPaths = [Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) + ";$installDir;$installDir/bin"
+    $newPaths = [Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) + ";$installDir"
```

Comment:
It seems like that the changes in this PR is not for fixing any issue but an improvement. I strongly suggest to hold on it to avoid some unexpected issues to block containerd plan.

Reply:
sure, we can keep this in review, and it's not a  fix as you said, as we use only "tar.gz" as the only source of containerd.
I tested with a "zip" file, we will have two set of containerd.exe, but we won't have this kind of package.